docker/Dockerfile

```diff
@@ -14,15 +14,15 @@ RUN chmod +x /usr/local/bin/install-php-extensions; \
         @composer \
     ; \
     # strip debug symbols from extensions to reduce size
-    strip --strip-debug /usr/local/lib/php/extensions/no-debug-non-zts-20190902/grpc.so; \
-    strip --strip-debug /usr/local/lib/php/extensions/no-debug-non-zts-20190902/xdebug.so;
+    find /usr/local/lib/php/extensions -name grpc.so -exec strip --strip-debug {} \;; \
+    find /usr/local/lib/php/extensions -name xdebug.so -exec strip --strip-debug {} \;;
```

Comment:
+1 for using find, that's what I wanted to use in the first place, but I guess I messed smth up, that's why it did not work.
However if we go for a more abstracted way, we could also simply strip the debug signals in all extensions, so any .so file in the extension directory. That should not hurt if there is nothing to strip, but can probably lower the size of other extensions as well (maybe we need others in the future).